charts/karpenter/templates/controller/deployment.yaml

```diff
@@ -32,6 +32,8 @@ spec:
       containers:
         - name: manager
           image: {{ .Values.controller.image }}
+          args:
+            - --default-instance-profile={{ required "controller.defaultInstanceProfile is required to be set" .Values.controller.defaultInstanceProfile }}
```

Comment:
I *think* the webhook needs all of this as well, since the defaulting logic happens there.

Reply:
Yes... it does look like the webhook needs this. Since we're already making a breaking change in the Helm chart, would it make sense to move these options out of the controller tree in the values? I'm thinking something like `options.defaultInstanceProfile`, `options.ClusterName`, and `options.clusterEndpoint`. Thoughts?